x-pack/filebeat/module/cisco/asa/test/additional_messages.log-expected.json

```diff
@@ -0,0 +1,2922 @@
+[
+    {
+        "cisco.asa.connection_id": "111111111",
+        "cisco.asa.destination_interface": "fw111",
+        "cisco.asa.mapped_destination_ip": "127.0.1.1",
+        "cisco.asa.mapped_destination_port": 53500,
+        "cisco.asa.mapped_source_ip": "127.0.1.1",
+        "cisco.asa.mapped_source_port": 53500,
+        "cisco.asa.message_id": "302013",
+        "cisco.asa.source_interface": "net",
+        "destination.address": "127.0.1.1",
+        "destination.ip": "127.0.1.1",
+        "destination.port": 53500,
+        "event.action": "firewall-rule",
+        "event.category": [
+            "network"
+        ],
+        "event.code": 302013,
+        "event.dataset": "cisco.asa",
+        "event.kind": "event",
+        "event.module": "cisco",
+        "event.original": "%FTD-6-302013: Built inbound TCP connection 111111111 for net:127.0.1.1/53500 (127.0.1.1/53500) to fw111:127.0.1.1/53500 (127.0.1.1/53500)",
+        "event.severity": 6,
+        "event.timezone": "-02:00",
+        "event.type": [
+            "info"
+        ],
+        "fileset.name": "asa",
+        "host.hostname": "dev01",
+        "input.type": "log",
+        "log.level": "informational",
+        "log.offset": 0,
+        "network.direction": "inbound",
+        "network.iana_number": 6,
+        "network.transport": "tcp",
+        "observer.egress.interface.name": "net",
+        "observer.hostname": "dev01",
+        "observer.ingress.interface.name": "fw111",
+        "observer.product": "asa",
+        "observer.type": "firewall",
+        "observer.vendor": "Cisco",
+        "related.ip": [
+            "127.0.1.1",
```

Comment:
The duplication of this value in this field is going to trigger some internal rules we have that try and ensure unique values on the `related.ip` entries. Not sure how simple it will be, but is there a way to dedup it?

Reply:
This issue here would happen only on our test data, but I have been working on trying to get some dedup script in painless going for related.* fields since it is quite a common issue. Will look into it.

Reply:
After more research I found that there is no real risk of these being duplicated, and I also saw that we currently don't handle duplication in most or any module.
I have updated the test data to be more realistic, but I will continue to investigate a better duplication handling possibility after this PR if possible.

Reply:
that makes sense, I'm fine with that